lib/request-wrapper.ts

```diff
@@ -132,45 +132,25 @@ export class RequestWrapper {
       this.axiosInstance.interceptors.request.use(
         (config) => {
           logger.debug('Request:');
-          try {
-            logger.debug(JSON.stringify(config, null, 2));
-          } catch {
-            logger.error(config);
-          }
-
+          logger.debug(config);
           return config;
         },
         (error) => {
           logger.error('Error: ');
-          try {
-            logger.error(JSON.stringify(error, null, 2));
-          } catch {
-            logger.error(error);
-          }
-
+          logger.error(error);
```

Comment:
I continued using logger.error() in this case instead of changing this to logger.debug(), because this is logging the request after some sort of error has occurred.
To be honest, I'm not quite sure what sort of error might cause trigger this error path while trying to log the request, because not even a connection-refused type error would trigger it (I tried :) ).

Reply:
I couldn't trigger this either. I suppose one possibility is in the case of multiple interceptors and one earlier in the chain rejects, but regardless IMO it doesn't impact the correctness of this fix.